From 6bb303a867bd8d0dd9c38a808b71f11cc6d82646 Mon Sep 17 00:00:00 2001
From: "Christian Jaeger (Rustaceans)" <ch@christianjaeger.ch>
Date: Sat, 21 Apr 2018 17:10:22 +0100
Subject: .xscreensaver 000: [PATCH 057/191] .xscreensaver: ok *hav* to let it put monitor to
 sleep

Not getting done otherwise ech

Also, use desktop background pic, enable more savers with those.
---
 .xscreensaver | 24 +++++++++++++-----------
 1 file changed, 13 insertions(+), 11 deletions(-)

diff --git a/.xscreensaver b/.xscreensaver
--- a/.xscreensaver
+++ b/.xscreensaver
@@ -1,5 +1,5 @@
 # XScreenSaver Preferences File
-# Written by xscreensaver-demo 5.36 for rustaceans on Sat Apr 21 14:37:54 2018.
+# Written by xscreensaver-demo 5.36 for rustaceans on Sat Apr 21 17:10:14 2018.
 # https://www.jwz.org/xscreensaver/
 
 timeout:	0:10:00
